dev/io.openliberty.session.6.0.internal/resources/io/openliberty/session60/internal/resources/SessionMessages.nlsprops

```diff
@@ -0,0 +1,30 @@
+###############################################################################
+# Copyright (c) 2022 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+#CMVCPATHNAME io.openliberty.session.6.0.internal/resources/io/openliberty/session60/internal/resources/SessionMessages.nlsprops
+#COMPONENTPREFIX SESN
+#COMPONENTNAMEFOR SESN Session
+#ISMESSAGEFILE TRUE
+#NLS_MESSAGEFORMAT_VAR
+#NLS_ENCODING=UNICODE
+
+# Note:
+#Last used error meessage code was SESN8502I
+#
+#Use 8600 - 8699 for Servlet 6.0 - EE10
+#
+# -----------------------------------------------
+cookie.attribute.name.null=SESN8600E: The attribute name is null.
+cookie.attribute.name.null.explanation=The attribute name cannot be null.
+cookie.attribute.name.null.useraction=Use a valid attribute name.
+# -----------------------------------------------
+cookie.attribute.name.invalid=SESN8601E: The attribute name contains invalid character.
+cookie.attribute.name.invalid.explanation=The attribute name contains separator characters that cannot be used for Cookie attributes.
```

Comment:
Change to: The attribute name contains at least one of the following separator characters, which cannot be used in the attribute name on the Cookie class: ( ) < > @ , ; : \ " / [ ] ? = { }